scripts/bootstrap.sh

```diff
@@ -172,26 +172,7 @@ fi
 
 echo "Checking for .git file to identify local installation; \"fatal: not a git repository\" is expected elsewhere"
```

Comment:
Can remove this comment (`echo "Checking for .git ...`) and the `git config --global` line. Those were added so the `git rev-parse` command you already removed would work. 

(That's part of why I'm happy to see this special case go. Too much magic to answer the question, is this a developer's system?)